cmd/celestia-appd/cmd/root.go

```diff
@@ -56,6 +56,9 @@ func NewRootCmd() *cobra.Command {
 	rootCommand := &cobra.Command{
 		Use: "celestia-appd",
 		PersistentPreRunE: func(command *cobra.Command, _ []string) error {
+			command.SetOut(command.OutOrStdout())
+			command.SetErr(command.ErrOrStderr())
```

Comment:
does this change all logs to go to the output instead of the error? if so, is a breaking change for things that collect logs?

Reply:
if I understand correctly, this is just for the prints. the logs are handled separatly:
```
	var logWriter io.Writer
	if strings.ToLower(serverCtx.Viper.GetString(flags.FlagLogFormat)) == tmcfg.LogFormatPlain {
		logWriter = zerolog.ConsoleWriter{Out: os.Stderr}
	} else {
		logWriter = os.Stderr
	}

	logLvlStr := serverCtx.Viper.GetString(flags.FlagLogLevel)
	logLvl, err := zerolog.ParseLevel(logLvlStr)
	if err != nil {
		return fmt.Errorf("failed to parse log level (%s): %w", logLvlStr, err)
	}

	serverCtx.Logger = ZeroLogWrapper{zerolog.New(logWriter).Level(logLvl).With().Timestamp().Logger()}
```

Rootul can correct me

Reply:
> does this change all logs to go to the output instead of the error?

Output that targets stderr will still go to stderr. This fixes an issue where commands that used Cobra's 

```go
// Println is a convenience method to Println to the defined output, fallback to Stderr if not set.
func (c *Command) Println(i ...interface{}) {
	c.Print(fmt.Sprintln(i...))
}
```

would previously go to stderr because stdout wasn't configured for the command. The `status` command is an example that used this Println method

Reply:
nice, makes sense

it might still be a bit confusing to anyone reading the code, as the logs are set to go to stderr in the sdk

Reply:
> as the logs are set to go to stderr in the sdk

huh that seems unexpected. I'll try running a node pre / post this change to double check that I didn't break something

Reply:
wow you're right. the SDK does send logs to STDERR. Before (and after) this change, `error.txt` contains logs from the node:

```bash
./scripts/single-node.sh > output.txt 2> error.txt
# output.txt contains output from the script 
# error.txt contains logs from the node which is unexpected
```
```